backend/src/main/resources/data.sql

```diff
@@ -109,7 +109,7 @@ INSERT INTO TIMESLOT (ID, END, START, SURVEY_ID)
 VALUES (18, '2020-02-20 02:22:20Z', '2020-02-20 20:20:20Z', 5);
 
 INSERT INTO TIMESLOT (ID, END, START, SURVEY_ID)
-VALUES (19, '2020-02-20 20:22:20Z', '2020-02-20 20:20:20Z', 6);
+VALUES (19, '2021-02-20 20:22:20Z', '2021-02-20 20:20:20Z', 6);
```

Comment:
We could make sure that the timeslot will always be in the future using something like:
`VALUES (19, CURRENT_TIMESTAMP + 365, CURRENT_TIMESTAMP + 365, 6);`

Reply:
Alright, I'll try that